src/node/rpc/test/frontend_test.cpp

```diff
@@ -807,6 +832,81 @@ TEST_CASE("MinimalHandleFunction")
   }
 }
 
+TEST_CASE("Restricted verbs")
```

Comment:
I am generally against unit tests that are in a loop because when one of them breaks it sometimes less straightforward to debug. Do you think the test would be too long if it wasn't done in a while loop?

Reply:
We could test a few specific cases rather than the full set, but it feels sensible to be systematic and get full coverage? In this case there are 34 verbs to test, which doesn't seem sensible to write out explicitly.

I hope the `INFO` line makes it obvious which loop iteration has failed, so a failure says `logged: PROBLEMATIC', and its trivial to locally comment out the loop and replace it with `verb = PROBLEMATIC`? And looping over all verbs has caught something in my refactor that would have been missed otherwise (my initial bit-masking only handled the first 32 due to a casting error, so this test failed on `UNLINK` after logging `UNLINK` as the first failing case).